### PR TITLE
Display jsonlint error in parsing .mention-bot file with syntax error

### PR DESCRIPTION
**Pull request by jeffreyleeon**
connects #188

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ var util = require('util');
 var schedule = require('./schedule.js');
 
 var GitHubApi = require('github');
+var jsonlint = require("jsonlint");
```
Comment (vjeux) [inline server.js]: nit: single quote

File: server.js [hunk 2]
```diff
@@ -166,12 +168,20 @@ async function work(body) {
     }).catch(function(e) {
       if (e instanceof SyntaxError && repoConfig.actions.indexOf(data.action) !== -1) {
         // Syntax error while reading custom configuration file
+        var errorLog = '';
+        try {
+          jsonlint.parse(e.repoConfig)
```
Comment (vjeux) [inline server.js]: Sweet!
image

**Comment by vjeux** [comment]
Awesome! Thanks, this is an awesome output. Please tweak the little quote and i'll merge it in :)

**Comment by hunkim** [comment]
@jeffreyleeon Very cool!

**Comment by vjeux** [comment]
YAY

**Pull request merged**